chrome/content/zotero-better-bibtex/cayw.coffee

```diff
@@ -102,7 +102,7 @@ class Zotero.BetterBibTeX.CAYW.CitationEditInterface
     # "label":"line","locator":"xx"
     citations = []
     for citation in @citation.citationItems
-      citation.label = 'page' if !citation.label && citation.locator
+      citation.label = 'p.' if !citation.label && citation.locator
```

Comment:
Doesn't mean we couldn't also (re)use a lookup inside the pandoc formatter. But this "if" just mends the Zotero picker, which is a bit whacky; it returns a locator label (kind, really) for every kind of locator you choose, except for "page". What that should display is a later case, it's more an identifier than a label at that point in the code.